drivers/wdm/audio/backpln/portcls/undoc.cpp

```diff
@@ -321,7 +313,7 @@ PcAddToPropertyTable(
     //UNICODE_STRING GuidBuffer;
 
 ASSERT(PropertyItem->Set);
-	//	RtlStringFromGUID(*PropertyItem->Set, &GuidBuffer);
+        //      RtlStringFromGUID(*PropertyItem->Set, &GuidBuffer);
```

Comment:
do not touch that line. It was commented anyway. Your patch won't need that change.

Reply:
Fixed.